webapp/src/types/common/globalState.ts

```diff
@@ -0,0 +1,5 @@
+import {GlobalState as OriginalGlobalState} from 'mattermost-redux/types/store';
+
+export interface GlobalState extends OriginalGlobalState {
+    views?: any;
```

Comment:
I'm thinking this will be more helpful with typescript

```suggestion
    views?: {
        rhs?: {
            rhsState: string;
            pluggableId: string;
        };
    };
```

Reply:
@mickmister Updated the PR with these changes